bin/rspec

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+#
+# This file was generated by Bundler.
```

Comment:
same here, why did this change?

Reply:
Rspec test within vs code stopped working for me. Adding those fixed the issue, so I added them with this PR. I can move them out if you prefer to do a bug story for it?

Reply:
no this is fine, was just curious